### ISSUE TITLE
Rejects empty TOML tables

### ISSUE THREAD

**Issue by real-or-random** [mikefarah/yq#1924]
**Describe the bug**

yq rejects  perfectly legal empty TOML tables 
Version of yq: v4.40.5

**Input Toml**

input.toml:
```
[emptyTable]
```

**Command**
```
yq -o=json '.' input.toml
```

**Actual behavior**
```
Error: bad file 'input.toml': error retrieving table [emptyTable] value: %!w(<nil>)
```

**Expected behavior**

Parsing works. https://toml.io/en/v1.0.0#table says: "Empty tables are allowed and simply have no key/value pairs within them."

**Comment by mikefarah** [comment]
Looking at other TOML -> JSON tools, it looks like empty tables should map to an empty _object_. I've updated your fix @elibroftw  with that in mind. Will be part of the next release.

**Comment by mikefarah** [comment]
Fixed in 4.40.7
